support.md

```diff
@@ -69,61 +69,41 @@ In addition to the [general guidelines](/support.html#CommunityGuidelines) above
 
 (_Credit to the [Fedora Mailing List Guidelines](http://fedoraproject.org/wiki/Communicate/MailingListGuidelines) as a source of ideas, under the Creative Commons Attribution-ShareAlike 3.0 Unported license_)
 
-## [libera.chat](https://libera.chat) {#IRClivechat}
-A great place to get real-time help with The Foreman is our IRC channel:
-
-* **#theforeman** on [libera.chat](https://libera.chat/)
+## ![Matrix icon](static/images/matrix.png) [Matrix.org live chat](https://riot.im/app/#/room/#theforeman:matrix.org)
```

Comment:
Might be better to use the more generic URL for the matrix rooms?

https://matrix.to/#/#theforeman:matrix.org

Reply:
These takes me through 2 more screen though, right? I like it mentions more matrix clients. Any other benefit? (I'm not opposed to it)

Reply:
The simple change would just be to use the new Element URL rather than the legacy Riot domain. The matrix.to share link lets the user access the room with the client of their choice, but it does involve a few extra clicks if they don't already have an app or web session ready to go.

Either way, a user can't preview the room/discussion until they have an account on some server. While linking directly to the room makes sense, using matrix.to makes sense from the Matrix community perspective of showcasing the decentralized nature of the platform. Whether or not that will just be more confusing for new non-Matrix users... I'll leave that to someone else to determine as I am a biased source.